.gitmodules

```diff
@@ -1,48 +1,64 @@
 [submodule "sorc/crtm"]
 	path = sorc/crtm
 	url = https://github.com/jcsda/crtm
+	branch = release/crtm_jedi_v2.4.1
 [submodule "sorc/femps"]
 	path = sorc/femps
 	url = https://github.com/jcsda/femps
+	branch = develop
 [submodule "sorc/fms"]
 	path = sorc/fms
 	url = https://github.com/jcsda/fms
+	branch = release-stable
 [submodule "sorc/fv3"]
 	path = sorc/fv3
 	url = https://github.com/jcsda/GFDL_atmos_cubed_sphere
+	branch = release-stable
 [submodule "sorc/fv3-jedi"]
 	path = sorc/fv3-jedi
 	url = https://github.com/jcsda/fv3-jedi
+	branch = develop
 [submodule "sorc/fv3-jedi-km"]
 	path = sorc/fv3-jedi-lm
 	url = https://github.com/JCSDA/fv3-jedi-linearmodel
+	branch = develop
 [submodule "sorc/gsibec"]
 	path = sorc/gsibec
 	url = https://github.com/GEOS-ESM/GSIbec
+	branch = develop
 [submodule "sorc/ioda"]
 	path = sorc/ioda
 	url = https://github.com/jcsda/ioda
+	branch = develop
 [submodule "sorc/iodaconv"]
 	path = sorc/iodaconv
 	url = https://github.com/jcsda-internal/ioda-converters
+	branch = develop
 [submodule "sorc/jedicmake"]
 	path = sorc/jedicmake
 	url = https://github.com/jcsda/jedi-cmake
+	branch = develop
```

Comment:
This one is a little confusing to me. Which is the newer one?  master or develop?

Reply:
`master` was the default GitHub repo branch name, it has since been replaced by `main`. But the branches that are synced are `develop` so that is the one you should always use.